### PR TITLE
Draft PR for GEOS-Chem input data.

### PR DESCRIPTION
**Pull request by yidant**
*Description of changes:*
Draft PR for GEOS-Chem input data. This PR adds geos-chem-input-data.yaml. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by msulprizio** [comment]
@yidant Pull request #2301 has now been merged. Feel free to rename your file to `geoschem-input-data.yaml` and sync with the any relevant info now in https://github.com/awslabs/open-data-registry/blob/main/datasets/geoschem-nested-input-data.yaml.

**Comment by yidant** [comment]
Thanks @msulprizio ! I've updated the yaml file name and its content!

File: datasets/geoschem-input-data.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - air quality
+  - chemistry
+  - atmosphere
+  - model
+License: https://geoschem.github.io/license.html
+Resources:
+  - Description: Top-level directory for all GEOS-Chem data.
+    ARN: unknown
+    Region: us-east-1
+    Type: S3 Bucket
```
Comment (msulprizio) [inline datasets/geoschem-input-data.yaml:26]: Can we add `Explore` here with the link to browse the bucket or do we not have a URL yet?

See [these lines](https://github.com/awslabs/open-data-registry/blob/cbb2a2c5d38ca4be9de3a8af1401efd38c63d8da/datasets/geoschem-nested-input-data.yaml#L27C1-L28C68) for an example.

File: datasets/geoschem-input-data.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - Title: GEOS-Chem Classic Quickstart Guide
+      URL: https://geos-chem.readthedocs.io/en/latest/getting-started/quick-start.html
+      AuthorName: GEOS-Chem Support Team
+      AuthorURL: http://geos-chem.org/support-team
+    - Title: GCHP Quickstart Guide
+      URL: https://gchp.readthedocs.io/en/latest/getting-started/quick-start.html
+      AuthorName: GEOS-Chem Support Team
+      AuthorURL: http://geos-chem.org/support-team
+  Tools & Applications:
+  Publications:
```
Comment (msulprizio) [inline datasets/geoschem-input-data.yaml:42]: Could you add links to the GCClassic and GCHP on the cloud papers? I think it's OK that these technically used the older S3 bucket.

```
    - Title: "Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model"
      URL: https://doi.org/10.1175/BAMS-D-18-0243.1
      AuthorName: Jiawei Zhuang et al.
    - Title: "Enabling high-performance cloud computing for Earth science modeling on over a thousand cores: application to the GEOS-Chem atmospheric chemistry model"
      URL: https://doi.org/10.1029/2020MS002064
      AuthorName: Jiawei Zhuang et al.
```

**Review by msulprizio — CHANGES_REQUESTED** [review]
Hi @yidant. I've requested some minor changes to the geoschem-input-data.yaml file here. 

Also tagging @yantosca and @lizziel for any feedback before this is merged in.

**Comment by yidant** [comment]
Thanks @msulprizio and @cstner! There're new updates made and ready for merge!

**Pull request merged**